test/testcafe/spec/EnrollWebauthnAuthenticator_spec.js

```diff
@@ -0,0 +1,28 @@
+import { RequestMock } from 'testcafe';
+import EnrollWebauthnAuthenticatorPageObject from '../framework/page-objects/EnrollWebauthnAuthenticatorPageObject';
+import xhrAuthenticatorEnrollWebauthn from '../../../playground/mocks/data/idp/idx/authenticator-enroll-webauthn';
+import xhrSuccess from '../../../playground/mocks/data/idp/idx/success';
+
+const mock = RequestMock()
+  .onRequestTo('http://localhost:3000/idp/idx/introspect')
+  .respond(xhrAuthenticatorEnrollWebauthn)
+  .onRequestTo('http://localhost:3000/idp/idx/challenge/answer')
+  .respond(xhrSuccess);
+
+fixture(`Enroll Webauthn Authenticator`)
+  .requestHooks(mock);
+
+async function setup(t) {
+  const enrollWebauthnPage = new EnrollWebauthnAuthenticatorPageObject(t);
+  await enrollWebauthnPage.navigateToPage();
+  return enrollWebauthnPage;
+}
+
+// This is the only test we can perfrom in testcafe as webauthn api is not available in the version of chrome/chromeheadless
```

Comment:
just curious do u know why? at least your local, testcafe using my current latest chrome. (wonder if there are some control parameter that testcafe send to webauthn)

Reply:
yeah seems like that. Even if thats the case I am testing all the stuff in my unit test since I have to mock some functions like isWebauthn available or is Edge etc.